corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -1,8 +1,35 @@
 'use strict';
-/* global Backbone, Marionette */
-hqDefine("cloudcare/js/formplayer/router", function () {
-    var utils = hqImport("cloudcare/js/formplayer/utils/utils");
-    var Router = Marionette.AppRouter.extend({
+hqDefine("cloudcare/js/formplayer/router", [
+    'underscore',
+    'backbone',
+    'backbone.marionette',
+    'cloudcare/js/formplayer/utils/utils',
+    'cloudcare/js/formplayer/app',
+    'cloudcare/js/formplayer/middleware',
+    'cloudcare/js/formplayer/apps/controller',
+    'cloudcare/js/formplayer/menus/collections',
+    'cloudcare/js/formplayer/menus/controller',
+    'cloudcare/js/formplayer/sessions/controller',
+    'cloudcare/js/formplayer/users/controller',
+    'cloudcare/js/formplayer/users/models',
+    'marionette.approuter/lib/marionette.approuter.min',    // for Marionette.AppRouter
+    'cloudcare/js/formplayer/sessions/api',     // for getSession
```

Comment:
was resolving conflict on staging and noticed, is this needed? I think `getSession` is defined within `router.js`

Reply:
This is about the `getSession` channel request that's inside the getSession function ([this](https://github.com/dimagi/commcare-hq/blob/615ebbdc732420c1dcb0817b2ae655da1968a46e/corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js#L98)), because the reply for that is defined in sessions/api.

Reply:
Ah, I always thought just importing where the channel is defined `cloudcare/js/formplayer/app` was enough. Thanks for explaining